app/src/main/java/org/wikipedia/theme/ThemeChooserDialog.kt

```diff
@@ -100,7 +99,7 @@ class ThemeChooserDialog : ExtendedBottomSheetDialogFragment() {
 
     override fun onStart() {
         super.onStart()
-        BottomSheetBehavior.from(requireView().parent as View).peekHeight = DimenUtil.roundedDpToPx(DimenUtil.getDimension(R.dimen.themeChooserSheetPeekHeight))
+        BottomSheetBehavior.from(requireView().parent as View).peekHeight = (DimenUtil.displayHeightPx * 0.75).toInt()
```

Comment:
We can have another set here to expand the bottom sheet by default.
```
BottomSheetBehavior.from(requireView().parent as View).state = BottomSheetBehavior.STATE_EXPANDED
```

Reply:
but Robin wants it to be at .75 of the screen cooltey.... if i set only the state then it might expand beyond .75 on small screens